google/cloud/aiplatform/jobs.py

```diff
@@ -1453,9 +1494,22 @@ def run(
                 restart_job_on_worker_restart=restart_job_on_worker_restart,
             )
 
+        if tensorboard:
+            v1beta1_gca_resource = (
```

Comment:
Is there some hidden coupling between Tensorboard and HyperparameterTuningJob?

Weird that this isn't made explicit in the protos.

Reply:
It's in the protos nested in the CustomJobSpec in HP Tuning Job:
https://github.com/googleapis/python-aiplatform/blob/master/google/cloud/aiplatform_v1beta1/types/hyperparameter_tuning_job.py#L108

https://github.com/googleapis/python-aiplatform/blob/aab9e589426331bfe7ac3f6efa97109e0bd0db0d/google/cloud/aiplatform_v1beta1/types/custom_job.py#L181